modules/opensearch/includes/opensearch.search.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * Implements handling of loans through OpenRuth.
+ */
+
+/**
+ * Get a list of material types from the Well.
+ */
+function opensearch_search_material_types() {
```

Comment:
Come to think of it - a connie_search would also be great for the aleph-development where @tripox is going to want some interim stub-implementation of the search-abstraction